kernel/src/main/jjtree/TagSelector.jjt

```diff
@@ -141,7 +141,7 @@ SKIP :
 
 TOKEN : /* TAG */
 {
-  <TAG: (["_","a"-"z","A"-"Z","0"-"9","#","$","%","^","*","+",".","-"])+>
+  <TAG: (["a"-"z", "A"-"Z", "0"-"9", "#", "$", "%", "^", "*", "+", ".", "-", "\"", "=", "_"])+>
```

Comment:
So we are not adding `:` after all?

Reply:
I'm still waiting for someone from the web-dev team to answer me if it may be helpful for them. Although, since it's blocking others' PR, we can move forward with this, and if necessary, I can open other PR and implement it.

Reply:
Ok